### PR TITLE
Sept 26th blogpost

### PR DESCRIPTION
**Pull request by eclarizio**
The 'B' blogpost to follow up Drew's 'A' post. Although, not quite as flashy, I admit.

### CODE REVIEW

**Comment by himdel** [comment]
@eclarizio the filename should contain more of the title, otherwise we may get url uniqueness (lack of) problems..

File: source/blog/2016-09-26-last-week-in-manage-iq.html.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+comments: true
+published: true
+---
+
+After [Drew's](https://github.com/d-m-u) wonderful 'A' [blog](http://manageiq.org/blog/2016/09/last-week-in-manageiq-absence-atomic-providers/), I, [Erik](http://github.com/eclarizio), will be guiding you through the 'B' blog! There were a lot of fun and exciting changes so lets get started!
+
+## The stats
+
+Last week, we merged [122 PRs][PRs merged last week], of which, [49 were enhancements][Enhancements merged last week]. In total, there were [414 commits!][Commits merged last week].
+
```
Comment (syncrou) [inline source/blog/2016-09-26-last-week-in-manage-iq.html.md]: @eclarizio - I see an extra `.` after the `!`

File: source/blog/2016-09-26-last-week-in-manage-iq.html.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+## Builders and BaseControllers! (Improved)
+
+### [Convert myco_tree to use TreeBuilder](https://github.com/ManageIQ/manageiq/pull/10904)
+
+[Zita](https://github.com/ZitaNemeckova) is continuing work on refactoring the way some of our left-side trees are built by using TreeBuilders. This will hopefully make it easier in the future if/when we need to make adjustments.
+
+### [Clean up Api::BaseController a bit](https://github.com/ManageIQ/manageiq/pull/11085)
+
+I'm a sucker for a good refactoring story, and [Tim](https://github.com/imtayadeway) has told a good one here. As with the tree builders, making things easier to adjust in the future is always a welcome change.
```
Comment (syncrou) [inline source/blog/2016-09-26-last-week-in-manage-iq.html.md]: @eclarizio - Haha! So many good refactoring stories out there. 😉

**Comment by syncrou** [comment]
@eclarizio - Great stuff!  Only a small comment in regards to punctuation.

**Comment by eclarizio** [comment]
@syncrou @himdel Issues fixed! Thanks.

**Comment by JPrause** [comment]
@Fryguy please merge for this week's blog LWIMIQ blog entry.

**Pull request merged**